i18n/en.json

```diff
@@ -798,6 +798,10 @@
     "id": "api.command_invite.user_already_in_channel.app_error",
     "translation": "{{.User}} is already in the channel."
   },
+  {
+    "id": "api.command_invite.user_not_in_team.app_error",
+    "translation": "{{.Username}} is not a member of the team."
```

Comment:
This results to:
<img width="306" alt="Screen Shot 2019-10-31 at 3 20 10 PM" src="https://user-images.githubusercontent.com/5334504/67927130-cce92680-fbf2-11e9-9ef4-b5bf160fca20.png">

I wonder if it should have `@` before the username.  Not sure if all but most of system messages have that convention so that it's clickable and can see details on profile popover.  @asaadmahmood Any thoughts before merging this PR?

Reply:
You're right, we should use an `@` when the username is being shown.

Reply:
Added `@` to the username. There are lots of other uses of username without `@` though.

Reply:
Thanks @asaadmahmood @iomodo!